README.md

```diff
@@ -8,34 +8,38 @@
 [![Docs](https://img.shields.io/badge/docs-doxygen-orange.svg)](https://www.gnuradio.org/doc/doxygen/)
 [![Packaging status](https://repology.org/badge/tiny-repos/gnuradio.svg)](https://repology.org/project/gnuradio/badges)
 
-GNU Radio is a free & open-source software development toolkit that 
-provides signal processing blocks to implement software radios. It can 
-be used with readily-available, low-cost external RF hardware to create 
-software-defined radios, or without hardware in a simulation-like 
-environment. It is widely used in hobbyist, academic, and commercial 
-environments to support both wireless communications research and real-world 
+GNU Radio is a free & open-source software development toolkit that
+provides signal processing blocks to implement software radios. It can
+be used with readily-available, low-cost external RF hardware to create
+software-defined radios, or without hardware in a simulation-like
+environment. It is widely used in hobbyist, academic, and commercial
+environments to support both wireless communications research and real-world
 radio systems.
 
-Please visit the GNU Radio website at https://www.gnuradio.org/ and the 
-wiki at https://wiki.gnuradio.org/. Bugs and feature requests are 
-tracked on GitHub's [Issue Tracker](https://github.com/gnuradio/gnuradio/issues). 
-If you have questions about GNU Radio, please search the **discuss-gnuradio** 
-mailing list [archive](https://lists.gnu.org/archive/html/discuss-gnuradio/), 
-as many questions have already been asked and answered. Please also 
-[subscribe](https://lists.gnu.org/mailman/listinfo/discuss-gnuradio) to 
+Please visit the GNU Radio website at https://www.gnuradio.org/ and the
+wiki at https://wiki.gnuradio.org/. Bugs and feature requests are
+tracked on GitHub's [Issue Tracker](https://github.com/gnuradio/gnuradio/issues).
+If you have questions about GNU Radio, please search the **discuss-gnuradio**
+mailing list [archive](https://lists.gnu.org/archive/html/discuss-gnuradio/),
+as many questions have already been asked and answered. Please also
+[subscribe](https://lists.gnu.org/mailman/listinfo/discuss-gnuradio) to
 the mailing list and post your new questions there.
 
 
 ## How to Install GNU Radio
 
 ### Prebuilt Binaries
 
-The recommended way to install GNU Radio on most platforms is using available binary package distributions. 
+The recommended way to install GNU Radio on most platforms is using available binary package distributions.
 
 The following command is for Debian, Ubuntu, and derivatives. Consult your distribution information to obtain the version of GNU Radio which is included.
 
     sudo apt install gnuradio
 
+For MacOS, install the latest release version of GNUradio with [homebrew](https://brew.sh)
```

Comment:
Wrong spelling/capitalization of "GNU Radio"

Reply:
fixed it